### PR TITLE
feat: convenient option to rollout new bento

### PR DESCRIPTION
**Pull request by frostming**
Signed-off-by: Frost Ming <me@frostming.com>

## Before submitting:

- [ ] Does the Pull Request follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary) naming? Here are [GitHub's
      guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on how to create a pull request.
- [ ] Does the code follow BentoML's code style, `pre-commit run -a` script has passed ([instructions](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#style-check-auto-formatting-type-checking))?
- [ ] Did you read through [contribution guidelines](https://github.com/bentoml/BentoML/blob/main/CONTRIBUTING.md#ways-to-contribute) and follow [development guidelines](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#start-developing)?
- [ ] Did your changes require updates to the documentation? Have you updated
      those accordingly? Here are [documentation guidelines](https://github.com/bentoml/BentoML/tree/main/docs) and [tips on writting docs](https://github.com/bentoml/BentoML/tree/main/docs#writing-documentation).
- [ ] Did you write tests to cover your changes?

### CODE REVIEW

File: src/bentoml/_internal/cloud/deployment.py [hunk 1]
```diff
@@ -197,8 +200,13 @@ def update(
         if bento is None:
             # NOTE: bento.repository.name is the bento.name, and bento.name is the bento.version
             # from bentocloud to bentoml.Tag concept
-            bento = f"{deployment_target.bento.repository.name}:{deployment_target.bento.name}"
+            bento = deployment_target.bento.repository.name
         bento = Tag.from_taglike(bento)
+        if bento.version in (None, "latest"):
```
Comment (sauyon) [inline src/bentoml/_internal/cloud/deployment.py:205]: Hm, if the user specifies `latest` shouldn't we get the latest bento always?

File: src/bentoml_cli/deployment.py [hunk 2]
```diff
@@ -114,10 +118,19 @@ def update(  # type: ignore
...
         """
-        res = client.deployment.update_from_file(
-            path_or_stream=file,
-            context=context,
-        )
+        if file is None and name is None:
+            raise click.BadArgumentUsage(
+                "Either --file or --name is required for update command"
+            )
+        if file is not None:
```
Comment (sauyon) [inline src/bentoml_cli/deployment.py:125]: We should maybe print a warning if name is set here.

**Comment by Haivilo** [comment]
Looks good. In this case I assume we need to run `bentoml push` before the `bentoml deployment update`?

**Comment by frostming** [comment]
> Looks good. In this case I assume we need to run `bentoml push` before the `bentoml deployment update`?

Yes, we do not want to make commands couple with each other. For instance, that will need to copy the flags to the command

**Review by aarnphm — APPROVED** [review]

**Pull request merged**